### ISSUE TITLE
Add seed without server restart

### ISSUE THREAD

**Issue by hakantunc** [webgme/webgme-setup-tool#80]
When a seed is added, the server needs to be restarted. Can it be possible to enable a seed without a restart.

**Comment by brollb** [comment]
When the WebGME reads the available seeds, it reads all the seed projects in the seed base paths. When you want to share a seed from a project, the new project will use the same seed base path. However, if all the seeds share a directory, then all the seeds will be shared (basically, an all-or-none situation).

To avoid this, the seeds are placed in their own directory. The benefit of this is that it allows users to share individual seed projects. However, this also means that the base paths need to be changed in the config to add the seed project to the project.

tl;dr - Not without a change to the webgme or the current design.
